### PR TITLE
ci: import shellcheck lint

### PR DESCRIPTION
**Pull request by kwanhur**
### What this PR does / why we need it:

1. shellcheck lint on action
2. fix shebang missing

test case log see it [here](https://github.com/kwanhur/apisix/actions/runs/1847641492)

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by tzssangglass** [comment]
hi @kwanhur , PLAT: https://github.com/apache/apisix/actions/runs/1847690178

It looks like this action is not available.

**Comment by leslie-tsang** [comment]
Hello there, plz refer to [PR #5364 comment](https://github.com/apache/apisix/pull/5364#pullrequestreview-792825733)
> By Apache's policy, only actions from GitHub or Apache can be used in the CI. So we have to use submodule.

**Comment by kwanhur** [comment]
ok, I’ll update it with submodule.


Kwanhur Huang
TL;DR

**Comment by leslie-tsang** [comment]
Hello there, Plz don't force push during the <ins>review</ins>.

Commit history allows reviewers to see what has changed since the last review, which is <ins>useful</ins> for them.

A forced push could <ins>**ruin**</ins> the review experience.

**Comment by kwanhur** [comment]
got it :-)

I have submit another pr#6340


Kwanhur Huang
TL;DR
